include/grpc++/security/auth_context.h

```diff
@@ -37,6 +37,7 @@
 #include <iterator>
 #include <vector>
 
+#include <grpc/grpc_security.h>
```

Comment:
I am not sure this will work. grpc++_unsecure has auth_context.h but not grpc_security.h